src/main/java/randoop/resource/generator/MethodListGen.java

```diff
@@ -0,0 +1,238 @@
+    Path unparsableSideEffectFreeFile = Paths.get(args[1], "JDK-sef-methods-unparsable.txt");
+
+    try {
+      List<String> nonDetMethods =
+          getAnnotatedMethodsFromJar(
+              classWorkingDirectory, AnnotationCategory.NON_DET, NONDET_ANNOTATIONS);
+      List<String> sideEffectFreeMethods =
+          getAnnotatedMethodsFromJar(
+              classWorkingDirectory, AnnotationCategory.SIDE_EFFECT_FREE, SEF_ANNOTATIONS);
+
+      // Randoop expects a list of omitmethods as regex.
+      if (nonDetMethods.size() > 0) {
+        System.out.println("Nondeterministic methods count: " + nonDetMethods.size());
+        try (BufferedWriter nonDetMethodWriter = Files.newBufferedWriter(nonDetFile, UTF_8)) {
+          nonDetMethodWriter.write(OMIT_METHODS_FILE_HEADER);
+          for (String method : nonDetMethods) {
+            nonDetMethodWriter.write("^" + Pattern.quote(method) + "$");
+            nonDetMethodWriter.newLine();
+          }
+          nonDetMethodWriter.flush();
+        }
+      } else {
+        System.out.println("No nondeterministic methods found. Not writing to file.");
+      }
+
+      // There are some fully qualified signatures that Randoop cannot parse.
+      // We will separate these into two files, the default (parsable) file and
+      // the non-parsable file.
+      if (sideEffectFreeMethods.size() > 0) {
+        System.out.println("Total side effect free methods count: " + sideEffectFreeMethods.size());
+        try (BufferedWriter sideEffectMethodWriter =
+                Files.newBufferedWriter(sideEffectFreeFile, UTF_8);
+            BufferedWriter unparsableSideEffectMethodWriter =
+                Files.newBufferedWriter(unparsableSideEffectFreeFile, UTF_8)) {
+          for (String fullyQualifiedMethodSignature : sideEffectFreeMethods) {
+            try {
+              signatureToOperation(
+                  fullyQualifiedMethodSignature,
+                  VisibilityPredicate.IS_ANY,
+                  new EverythingAllowedPredicate());
+              sideEffectMethodWriter.write(fullyQualifiedMethodSignature);
+              sideEffectMethodWriter.newLine();
+            } catch (RandoopUsageError e) {
+              System.err.println("Not parsable: " + e.getMessage());
+              unparsableSideEffectMethodWriter.write(fullyQualifiedMethodSignature);
+              unparsableSideEffectMethodWriter.newLine();
+            }
+          }
+          sideEffectMethodWriter.flush();
+        }
+      } else {
+        System.out.println("No side effect free methods found. Not writing to file.");
+      }
+    } catch (IOException e) {
+      e.printStackTrace();
+      System.exit(1);
+    }
+  }
+
+  /**
+   * Returns the annotated methods from the the given jar.
+   *
+   * @param jarFile JDK .jar file path
+   * @param annotationCategory type of annotation we are parsing for
+   * @param annotations list of annotations
```

Comment:
This is not descriptive of what the formal parameter is, means, or is used for.  This just restates the type, which is already explicit in the source code.